src/dymoprint/lib/unicode_blocks.py

```diff
@@ -5,25 +5,33 @@
 LH = "▄"
 FB = "█"
 NB = "\xa0"
+LS = "░"
 
 assert UH == "\N{UPPER HALF BLOCK}"
 assert LH == "\N{LOWER HALF BLOCK}"
 assert FB == "\N{FULL BLOCK}"
 assert NB == "\N{NO-BREAK SPACE}"
+assert LS == "\N{LIGHT SHADE}"
 
+BLACK_PIXEL = (255, 255, 255, 255)
+WHITE_PIXEL = (0, 0, 0, 255)
+TRANSPARENT_PIXEL = (0, 0, 0, 0)
```

Comment:
What are transparent pixels for? Probably some explanation of this and `LS` should be added to the code as comments.